infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/TrackLayout.kt

```diff
@@ -279,6 +279,11 @@ data class TrackNumberAndChangeTime(
     val changeTime: Instant,
 )
 
+data class SwitchesAtEnds(
+    val start: DomainId<TrackLayoutSwitch>?,
+    val end: DomainId<TrackLayoutSwitch>?,
```

Comment:
Näitä vissiin välitetään frontille asti, niin voisiko nää rajata `IntId`:iksi?

Reply:
Joo, castihan siihen pitää laittaa mutta kyllähän se tarkoitusta selkeyttää.